string name
float64[] wrench
string[] vertex_frames
float64[] vertex_weights
bool contact_state
